packages/cli/src/command.ts

```diff
@@ -50,6 +50,27 @@ export const SYNC_OPTION_FLAG = Flags.integer({
   },
 })
 
+const FLAGS = [
+  '-o',
+  '--output',
+  '-k',
+  '--key',
+  '-c',
+  '--ceramic',
+  '-e',
+  '--common-embeds',
+  '--id-only',
+  '--table',
+]
+
+export function getArrayArg(argv: Array<string>, flags: CommandFlags): Array<string> {
```

Comment:
This logic seems very prone to error as we change flags in the different commands, could you try using OCLIF's arguments options or even leveraging the internal parser to get a better to extract the relevant inputs please?
Looking at the parser types here: https://github.com/oclif/core/blob/main/src/interfaces/parser.ts it seems the `ParsingToken` could be useful for example?